gcloud/monitoring/_helpers.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for shared behavior."""
+
+
+def _format_timestamp(timestamp):
```

Comment:
This doesn't exist anywhere else? 

And we can't accept timezone aware timestamps? Seems like that's a mistake... If we know how to accept UTC only if we specify that it's UTC by putting `Z` at the end, then we should know how to accept any timezone and do the basic math, right?

That is, shouldn't both of these be valid? And if not, shouldn't the service *only* accept timezone *unaware* values based on the assumption that they're all UTC?

```python
>>> now.isoformat()
'2016-07-30T13:31:34.190128-04:00'
>>> now.astimezone(pytz.utc).isoformat()
'2016-07-30T17:31:34.190128+00:00'
```